### DISCUSSION TITLE
Add the ability to turn multiple words into one tag

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#30083] [Ideas]
When making a beatmap, you are also asked to add tags to it. 
According to the wiki(https://osu.ppy.sh/wiki/en/Client/Beatmap_editor/Song_setup), tags are "Keywords for search, separated by spaces. May contain anything relevant to the beatmap or the song, such as the album name, guest mappers' names, or genre."
However, they are  never used for searching. This is because of the inability to combine different tags into one big tag.
Let's take Vanitas (https://osu.ppy.sh/beatmapsets/2095177#osu/4393066) as an example, since that is a map of the highest of standarts.
Here we have tags like "grand" "finals" "tiebreaker" or "featured" "artist"  which should instead be "grand finals tiebreaker" and "featured artist". Sure, the tag "tiebreaker" could maybe be used on it's own, but the rest is just useless.
Since tags in their current state are a completely useless and therefore not used thing, I propose to give mappers the ability to put multiple words as a single tag so that tags will actually serve their function in the future.

**Comment by Tenexxt** [comment]
There will be a tag "rework"(?) That will make tags more useful overall, and will also allow you for sorting map-types (aim, speed, etc.) That the players would choose, so this could (personally I think should) be added with the rework

**Reply by ghost** [reply]
good idea, I will try to find a person that works on that rework and get into contact with them.

**Comment by cihe13375** [comment]
I tried searching "grand finals tiebreaker" (no quotes) and this map (along with gftb in other tournamants) does appear.

**Reply by ghost** [reply]
Well sure, but it makes using tags for searching impossible. As you stated, you gotta go back to the beatmap listing and then search with the tag. However, you should be able to just click the tag on the beatmap page itself to ba able to search for it. But like in your example, you can only search for each word individually when you click a tag, which makes this function completely useless. However, they added this function to be used but right now it cannot do it's job. Furthermore, what about more complex tags. Sure, grand finals tiebreaker is a simple tag, but let's say you want to search for music geres. In the case of Vanitas, we have "progressive" "metal" which is supposed to be "progressive metal". Sure, if you know that, then that's all good, but most players have no clue about music genres. So they will think that "progressive" and "metal" are two separate genres of music. So I still think that my idea should be implemented.
